ajax/ajax.md

```diff
@@ -11,24 +11,25 @@ About 4 hours
 
 ### Prerequisites
 - [JavaScript Lessons](/javascript)
-- [jQuery Lesson](/jquery/jquery.md)
-- [API/JSON Lesson](/api/apis-and-json.md)
+- [jQuery Lesson](/jquery/jquery)
+- [API/JSON Lesson](/api/apis-and-json)
```

Comment:
Why did you removed the extension ?

Reply:
The file works without the extension. Should I add it back ?

Reply:
Yeah, add them back.